### ISSUE TITLE
When redpiler running, different results appeared

### ISSUE THREAD

**Issue by LimXu1** [MCHPR/MCHPRS#157]
I run same program on my "cpu", same data input, but different results come out.
as shown bellow, line "a" runs with redpilar running.
line "c" runs with redpilar stop.
line "b" runs with redpilar stop (rtps 20) for a while and then switch to rtps 1000 with redpilar running. 
Only line "c" meets expectations.
Additionally, I run this program only by "radv 10" command, finally, line c come out which is correct.
![image](https://github.com/user-attachments/assets/5f0d350c-a02b-42a7-a615-19f78180447c)

**Comment by LimXu1** [comment]
BTW, The direction of pixel generation is as follows.
<img width="827" alt="image" src="https://github.com/user-attachments/assets/eda3d8b9-3fb0-454b-bf6b-037ee66d5da6">

**Comment by StackDoubleFlow** [comment]
Is line "c" also the behavior in vanilla?

**Comment by LimXu1** [comment]
yes, it is. Line "c" runs with "rtps 20" and the redplier status shows as "stopped  1".
more test information as shown bellow:
This is run with "rtps 20", redplier status: "stopped  1"
![image](https://github.com/user-attachments/assets/5d897cb4-b30a-47f4-a36c-e4cd28f98840)
This is run with "rtps 100", redplier status: "running  1"
![image](https://github.com/user-attachments/assets/03d27994-7ee2-45f9-9800-da71eaa00216)

**Comment by commanderguy3001** [comment]
would you mind uploading the schematic somewhere so we can reproduce it?

**Comment by StackDoubleFlow** [comment]
To clarify, running without redpiler is not the same as running in vanilla

Agreed that we need the schematic file to look into this further

**Comment by LimXu1** [comment]
[temp0.zip](https://github.com/user-attachments/files/16738709/temp0.zip)
password has sent to your email @StackDoubleFlow

**Comment by IUM259** [comment]
Hi, what is the password ?

**Comment by LimXu1** [comment]
Hi, password: limxuxcl

获取Outlook for Android<https://aka.ms/AAb9ysg>
________________________________

**Comment by BramOtte** [comment]
Please do also test your build on a single player world, when I tested it in vanilla (with carpet mod to tell when it got stuck)
 it seemed to get stuck in this state...
It might very well be that your build doesn't even work in vanilla minecraft or that it is locational, if the build does work in vanilla make sure to also test it in a different location
![temp0-MCJE-screen](https://github.com/user-attachments/assets/059ce7c8-152e-48b9-b27f-85f961b6f026)

**Comment by LimXu1** [comment]
Hi, thanks for your support, I do further test about this issue, I found that the repeater behaves differently in situation of "redpiler stopped" and "redpiler running" . I suspect this is related to the redpiler status.


Here is the test video :https://youtu.be/4WLlUvoiZ8U;
Here is the plot copy: [world.zip](https://github.com/user-attachments/files/19719136/world.zip)
